### PR TITLE
fs-1968-score-history

### PR DESCRIPTION
**Pull request by gio-karunakaran**

### CODE REVIEW

File: db/queries/scores/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # flake8: noqa
-from .queries import get_latest_score_for_application_sub_crit
-from .queries import create_score_for_application_sub_crit
+from .queries import get_scores_for_app_sub_crit
+from .queries import create_score_for_app_sub_crit
```
Comment (Adam-W1) [inline db/queries/scores/__init__.py:3]: I dont think we need these __init__ files for 'queries/scores' and 'queries/assessment_records'

File: db/queries/scores/queries.py [hunk 2]
```diff
@@ -16,34 +16,46 @@
...
+            Score.application_id == application_id,
+            Score.sub_criteria_id == sub_criteria_id
+            ).order_by(Score.date_created.desc())
+    else:
+        stmt = select(Score).where(
+            Score.application_id == application_id,
+            Score.sub_criteria_id == sub_criteria_id
+            ).order_by(Score.date_created.desc()).limit(1)
+
+    score_rows = db.session.scalars(stmt)
```
Comment (Adam-W1) [inline db/queries/scores/queries.py:42]: neat ✨

File: api/routes/score_routes.py [hunk 3]
```diff
@@ -2,29 +2,30 @@
...
 
 
-def latest_score_for_application_sub_criteria(
-    application_id: str, sub_criteria_id: str
+def get_score_for_application_sub_criteria(
+    application_id: str, sub_criteria_id: str, score_history: bool = False
 ) -> Dict:
-    """latest_score_for_application_sub_criteria Function 
+    """get_score_for_application_sub_criteria Function 
     used by the get endpoint `/applications/{application_id}/
```
Comment (adamdavies1) [inline api/routes/score_routes.py:15]: does score_history need to be added to this?

File: db/queries/scores/__init__.py (hunk 1 above)
Comment (gio-karunakaran) [reply db/queries/scores/__init__.py:3]: your right! i think there outer __init __ file handles it good spot! :D

**Review by tferns — APPROVED** [review]
Nice work on this!

File: db/queries/assessment_records/__init__.py [hunk 4]
```diff
@@ -1,10 +0,0 @@
-# flake8: noqa
-from .queries import bulk_insert_application_record
-from .queries import find_answer_by_key_runner
-from .queries import get_metadata_for_fund_round_id
-
-__all__ = [
-    "get_metadata_for_fund_round_id",
-    "bulk_insert_application_record",
-    "find_answer_by_key_runner",
-]
```
[the remark this answers is not in the document]
Comment (gio-karunakaran) [reply db/queries/assessment_records/__init__.py:10]: we're treating all of the queries folder as a single package now, @Adam-W1 spotted we were already including the methods we need in the __init__ file of the query folder 👍🏾

**Pull request merged**